dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/deprecated/tracker/TrackedEntityControllerTest.java

```diff
@@ -89,19 +112,92 @@ public void setUp() throws BadRequestException, IOException {
             null,
             null,
             null,
-            null,
+            fileResourceService,
             trackerAccessManager,
             null,
             null,
             new TrackedEntityInstanceStrategyImpl(
                 trackedEntityInstanceSyncStrategy, trackedEntityInstanceAsyncStrategy),
-            userService);
+            userService,
+            config);
 
     mockMvc = MockMvcBuilders.standaloneSetup(controller).build();
+  }
+
+  @Test
```

Comment:
Is the only difference between both these tests is the content type "image/png" vs "file/png"? If yes, could we use ParameterizedTest instead?

Reply:
Not just this.
`/image` needs to mock the input stream 
`when(fileResourceService.getFileResourceContent(fileResource))
        .thenReturn(new FileInputStream(file));`
 while `/file` doesn't need it